CHANGELOG.rst

```diff
@@ -0,0 +1,10 @@
+Changelog
```

Comment:
Format of file should be same as in nereid https://github.com/openlabs/nereid/blob/develop/CHANGELOG